content/ordering.ts

```diff
@@ -20,6 +20,7 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
       name: 'Getting Started',
       description: "Start here if you're new to competitive programming.",
       items: [
+        'using-this-guideee',
```

Comment:
for some reason, this change causes the build to run indefinitely:

https://vercel.com/cpinitiative/usaco-guide/BfLuuWZ7fGUuux7wMFrDkiiLpEmy

is this expected? @thecodingwizard

Reply:
if I do `yarn run build` locally it stalls indefinitely at:

```
[==                          ]   529.033 s 50/665 8% Building static HTML for pages
```

Reply:
:0 it's not expected, and I don't really know why this happens :(